### PR TITLE
update the creating-tic page

### PR DESCRIPTION
**Pull request by jay4peace**
### Change Description ###

- update the creating-tic page

### CODE REVIEW

File: docs/platform/developing/creating-tic.md [hunk 1]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+   * Easy
+     * Select predefined set of images
+     * Click `Apply changes` button
+     ![Setting-Easy](./media/tic-ui.png)
+
+   * Advanced
+     * Select `Archtecture`
+     * Import your own `Recipe`, `Repository` and `Package` (optional)
+     * Click `Apply changes` button
+     ![Setting-Advanced](./media/tic-settings-advanced.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:50]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 2]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+
+#### Packages
+   * If you select `Easy` mode, in Settings stage, you can see `Summary` of the image to be generated.
+      ![Packages-Summary](./media/tic-packages-summary.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:54]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 3]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+
+   * If you select `Advanced` mode, in Settings stage, you can see `Details` tab.
+   ![Packages-Details](./media/tic-packages-details.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:57]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 4]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+     * Select `packages` to be included in the image to be create.
+   * Click `Create` button
+   ![Packages-Create](./media/tic-packages-create.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:60]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 5]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+     * Enter an Image name
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:61]:
```suggestion
     * Enter an image name
```

File: docs/platform/developing/creating-tic.md [hunk 6]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+     * Check `Generate SDK` checkbox, if you want to generate SDK package.
+   * Click `Create` button again.
+
+#### Jobs
+   * You can see the list of image creation jobs and their progress.
+   ![Jobs-Progress](./media/tic-jobs-progress.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:67]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 7]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+   * Click `X` button if you want to cancel the job.
+
+#### Images
+   * You can see the list of images
+   ![Images-List](./media/tic-images-list.png)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:72]: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

File: docs/platform/developing/creating-tic.md [hunk 8]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+   * Click the link of the Image or SDK to download them.
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:73]:
```suggestion
   * Click the link of the image or SDK to download them.
```

File: docs/platform/developing/creating-tic.md [hunk 9]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+
+To flash the generated image by TIC to a target device for verification, see [Flashing an Image to RPI](flashing-rpi.md).
+
+To use the generated SDK to develope applications with Tizen Studio, see [Configuring the Package Manager](../../application/tizen-studio/setup/advanced-configuration.md) and [Configurable SDK](../../application/tizen-studio/configurable-sdk/configurable-sdk.md)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:77]:
```suggestion
To use the generated SDK to develope applications with Tizen Studio, see [Configuring the Package Manager](../../application/tizen-studio/setup/advanced-configuration.md) and [Configurable SDK](../../application/tizen-studio/configurable-sdk/configurable-sdk.md).
```

File: docs/platform/developing/creating-tic.md [hunk 10]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+     * Check `Generate SDK` checkbox, if you want to generate SDK package.
+   * Click `Create` button again.
+
+#### Jobs
+   * You can see the list of image creation jobs and their progress.
+   ![Jobs-Progress](./media/tic-jobs-progress.png)
+   * Click `X` button if you want to cancel the job.
+
+#### Images
+   * You can see the list of images
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:71]:
```suggestion
   * You can see the list of images.
```

File: docs/platform/developing/creating-tic.md (hunk 5 above)
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:61]:
```suggestion
     * Enter an Image name.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @jay4peace, I have reviewed the file in this PR and made some correction suggestions.

Please check and incorporate them.

**Comment by jay4peace** [comment]
@omar-srbd I updated docs and images resources.

**Comment by jay4peace** [comment]
This page should be open with Tizen 8.0 release.
Please consider it.

File: docs/platform/developing/creating-tic.md [hunk 11]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+
+#### Settings
+   * Easy
+     * Select predefined set of images
+     * Click `Apply changes` button
+     ![Setting-Easy](./media/tic-ui.png)
+
+   * Advanced
+     * Select `Architecture`
+     * Import your own `Recipe`, `Repository & Package` (optional)
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:48]:
```suggestion
     * Import your own `Recipe, Repository & Package` (optional)
```

File: docs/platform/developing/creating-tic.md [hunk 12]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+     * Click `Apply changes` button
+     ![Setting-Advanced](./media/tic-settings-advanced.png)
+
+#### Packages
+   * If you select `Easy` mode, in Settings stage, you can see `Summary` of the image to be generated.
+      ![Packages-Summary](./media/tic-packages-summary.png)
+
+   * If you select `Advanced` mode, in Settings stage, you can see `Details` tab.
+   ![Packages-Details](./media/tic-packages-details.png)
+     * Select `packages` to be included in the image to be create.
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:58]:
```suggestion
     * Select `packages` to be included in the image that will be created.
```

File: docs/platform/developing/creating-tic.md [hunk 13]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+   * Click `Create` button
+   ![Packages-Create](./media/tic-packages-create.png)
+     * Enter an image name
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:61]:
```suggestion
     * Enter an image name.
```

**Review by omar-srbd — COMMENTED** [review]
@jay4peace, please check the suggestions.

File: docs/platform/developing/creating-tic.md [hunk 14]
```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
...
+      mkdir -p $HOME/tic-artifacts
       docker login -u [DockerHub_Username] -p [DockerHub_Password]
       docker-compose pull
       docker-compose up -d
    ```
 
 ### Connect to the TIC web-service
 
-Open the web browser and enter `http://127.0.0.1:8088` in the address bar. The TIC web UI screen appears:
+   * Open the web browser and enter `http://127.0.0.1:8088` in the address bar. The TIC web UI screen appears:
```
Comment (omar-srbd) [inline docs/platform/developing/creating-tic.md:35]:
```suggestion
```

**Review by omar-srbd — COMMENTED** [review]
@jay4peace, please check the suggestion.

**Review by omar-srbd — APPROVED** [review]
@jay4peace, thanks for incorporating the changes.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**